styles/components/_buttons.scss

```diff
@@ -325,6 +325,30 @@
   }
 }
 
+.btnFlx {
+  // see styleSheet.html for example usage.
+  // use for buttons in the new 'flex' style with more padding and use css flex.
+  @extend .btn;
+  padding: 25px;
+
+  border-radius: 0px;
+  border-top: 1px solid;
+
+  & + a {
```

Comment:
I think this should be `& + .btnFlx`, in case the sibling isn't an anchor tag. And to protect against siblings
that for some reason are anchor tags but not btnFlx.